src/pages/about.js

```diff
@@ -7,11 +7,83 @@ const AboutPage = ({data, location}) => {
 
   return (
     <Layout>
-      <SEO title="About :: What does it take to decarbonize your state?" />
+      <SEO title="About :: Decarbonize My State" />
       <div className="container">
-        <div className="border-bottom">
-          <h1 id="main" className="py-2">Overview</h1>
-          <p>About this app</p>
+        <div className="about">
+          <h1 id="main border-bottom" className="py-4">About Decarbonize My State</h1>
+          <p>We have 28 years to reduce worldwide emissions to zero to avoid the worst impacts of climate change. But how do we get there and what can I do to help?</p>
+          <p><strong>Decarbonize My State </strong>is a website built to answer that question and galvanize US residents to take direct and personal action to fight climate change. Inspired by a recent talk by Juan-Pablo Velez titled<a href="https://chihacknight.org/events/2022/01/11/juan-pablo-velez.html"> “What does it actually take to decarbonize a state?</a>” and the book<a href="https://www.rewiringamerica.org/electrify-the-book"> “Electrify, by Saul Griffith from Rewiring America</a>”, we've gathered data on climate pollution and its sources for each state to make convincing arguments for society-wide electrification and investment in renewable energy like wind and solar.</p>
+          <p>Our goals for this website are to help people working on this project and our immediate peers and neighbors to:</p>
+          <ul>
+            <li>Understand what has to happen to decarbonize each US state, and what policies need to be passed to drive those actions</li>
+            <li>Understand the highest impact ways individuals can help by</li>
+            <li>Electrifying their own fossil fuel infrastructure</li>
```

Comment:
I think the tense is changing between the list items - I'd change the last three to remove the `-ing`, so just "Electrify their own..." "Join political...".

Reply:
+1 to this. Maybe also for the first item, Learn instead of understand to avoid repetition